packages/manager/src/features/OneClickApps/oneClickApps.ts

```diff
@@ -862,6 +863,28 @@ export const oneClickApps: OCA[] = [
     summary: 'Free, open source video conferencing and communication platform.',
     website: 'https://jitsi.org/',
   },
+  {
+    alt_description:
+      'Open source video conferencing cluster, alternative to Zoom.',
+    alt_name: 'Video chat and video conferencing cluster',
+    categories: ['Media and Entertainment'],
+    colors: {
+      end: '949699',
+      start: '1d76ba',
+    },
+    description: `Secure, stable, and free alternative to popular video conferencing services. This app deploys four networked Jitsi nodes.`,
+    logo_url: 'jitsi.svg',
+    name: 'Jitsi',
```

Comment:
```suggestion
    name: 'Jitsi Cluster',
```

 I'm seeing an issue with the Jitsi app details drawer because of the name mismatch. The drawer loads once I update this to "Jitsi Cluster". 

![Screenshot 2024-04-23 at 2 10 06 PM](https://github.com/linode/manager/assets/114685994/642b0f2f-1967-4916-a5eb-fd1d0bc3d5a1)

Reply:
Looking good now with this change made in [a09af19](https://github.com/linode/manager/pull/10382/commits/a09af195b6c0e9eb99a5fe105a3710a736f96a30).

![image](https://github.com/linode/manager/assets/114685994/e70fcdfe-10da-4c5f-9afc-024a2ad6d7c7)